src/components/Blocking/SpecificIssue.js

```diff
@@ -68,7 +68,7 @@ const styles = StyleSheet.create({
     marginBottom: 19,
     fontFamily: fonts.inter[500],
     fontSize: 16,
-    color: COLORS.white2
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect the visibility and contrast of the text against its background, as it might impact user experience.
```diff
-    color: COLORS.white2
+    color: COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->